rfc006-release-planning-mgt.md

```diff
@@ -0,0 +1,143 @@
+
+- For technical writers, creating formal releases makes it easy to ensure they're always working from the latest stable/approved versions of templates.
+- For integrators (i.e. those who might create solutions based on our work), well-versioned releases make it easy to ensure (backward-)compatibility.
+- For project members, it aligns expectations on when to target tasks for completion, and way to ensure only updates of the appropriate value and quality are distributed to users.
+
+The Good Docs Project has grown to contain multiple sub-initiatives ranging from the templates (arguably the core of the project's activities at time of writing), to tooling aimed at technical writers, to useful content in the form of example documentation and blog posts. Given the project's increasing membership across multiple time zones, implementing a standard process through which the Project Steering Committee and Working Groups can define and manage releases will align expectations and better enable monitoring of progress.
+
+
+## Definition of a Release of The Good Docs Project
+
+As an organization with multiple sub-projects (some of which arguably have their own sub-projects) working with different cadences and outputs, we'll define for the purposes of the process this RFC describes what constitutes a "release" from The Good Docs Project. It's been noted that The Good Docs Project will produce two types of outputs:
+
+- "Product Oututs," or assets such as templates that will actually be used by our core audience in their own documentation efforts, and;
+- "Non-Product Outputs," which includes thought leadership, whitepapers, and other content that may inform documentation efforts but will not be used directly.
+
+After discussion among the PSC and community, we're defining a Release as **a snapshot of Product Outputs as agreed upon by the PSC and Working Groups**. At the time of writing, the following outputs are considered Product Outputs of TGDP:
+
+- The Templates collection
+- The Doc Tools initiatives
+- The Chronologue project, by virtue of its inclusion as examples to both the above
+
+Note however that while a given Release from The Good Docs Project may not include any *updated* material as part of a Product Output, its current version will still be included as part of any release packages that are creation. For example, the Doc Tools Working Group may release a new version of that software as part of a Release implementing a new feature. But the Chronologue WG's content included with that software need not be updated at all, unless the new feature specifically requires it (e.g. it makes use of a new metadata field).
+
+
+## Roles and Responsibilities
+
+The main participants in the Release Planning Process are as follows:
+
+- Project Steering Committee: The PSC will collectively create goals for releases, and 

```

Comment:
```suggestion
- Project Steering Committee: The PSC will review and approve proposed releases. 
```

I suggest that the role of the PSC is oversight and review. Sure, PSC members can suggest goals and dates to the release manager, but I suggest that it is the release manager's responsibility to drive this.

The work of selecting a date, setting a schedule, selecting a quality standard, is something that can be set up best by one person rather than a committee.